client/src/components/Music.jsx

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+import $ from 'jquery';
+import SongList from './SongList.jsx'
+import styled from 'styled-components';
+
+
+class Music extends React.Component {
+  constructor(props) {
+  super(props);
+  this.state = {
+    currentTrack: "",
+    play: false,
+    pause: true,
+    album:[],
+    artist: "",
+    audio: new Audio(""),
+    url: "",
+    time: "0:00",
+    intervalId: null,
+    albumTitle:"",
+    sliderValue: 0,
+    description:""
+  }
+
+  this.playTrack = this.playTrack.bind(this)
+  this.get = this.get.bind(this)
+  this.changeTrack = this.changeTrack.bind(this)
+  this.volumeDown = this.volumeDown.bind(this)
+  this.volumeUp = this.volumeUp.bind(this)
+  this.currentTrackTime =this.currentTrackTime.bind(this)
+  this.handleSlider = this.handleSlider.bind(this)
+}
+componentDidMount(){
+  this.get();
+  var intervalId = setInterval(this.currentTrackTime, 1000);
+  this.setState({intervalId: intervalId});
+}
+componentWillUnmount() {
+  // use intervalId from the state to clear the interval
+  clearInterval(this.state.intervalId);
+}
+currentTrackTime(){
+  var songTime =(parseInt(((this.state.audio.currentTime/60)-(parseInt(this.state.audio.currentTime/60)))*60));
+  var seconds = songTime < 10? "0"+ songTime: songTime
+
+  this.setState({
+    time: (parseInt(this.state.audio.currentTime/60))+":"+ seconds,
+    sliderValue: this.state.audio.currentTime
+  })
+}
+
+playTrack(){
+  if (this.state.play === false) {
+    this.setState({ play: true, pause: false })
+    this.state.audio.play();
+  } else {
+    this.setState({ play: false, pause: true })
+    this.state.audio.pause();
+  }
+}
+
+volumeDown(){
+    this.state.audio.volume -= .1
+  }
+volumeUp(){
+    this.state.audio.volume += .1
+  }
+
+get(){
+  $.get('/media/1', (data) => {
+    this.setState({
+      artist:data[0].artist,
+      album: data[0].album,
+      albumTitle: data[0].albumTitle,
+      currentTrack: data[0].album[0].track,
+      audio: new Audio(data[0].album[0].url),
+      url:data[0].album[0].url,
+      time: this.state.audio.currentTime,
+      description: data[0].artistDescription
+    })
+  })
+}
+handleSlider(e){
+  this.state.audio.currentTime = e.target.value;
+  // this.setState({
+  //     sliderValue: this.state.audio.currentTime
+  //   });
+
+}
+
+changeTrack(e){
+  this.state.audio.pause();
+  this.setState({
+    play: false,
+    pause: true,
+    currentTrack: this.state.album[e.target.id].track,
+    audio: new Audio(this.state.album[e.target.id].url),
+    url:this.state.album[e.target.id].url
+  })
+
+
+}
+
+
+render() {
+
+  return (
+    <Wrapper>
+      <br></br>
+      <Title>{this.state.albumTitle}<br></br>
+        <div style = {{fontSize: "20px"}}>By <Links>{this.state.artist}</Links></div>
+      </Title>
+      <MediaContainer>
```

Comment:
Good work splitting all of these into different components. Should be easier to navigate moving forward.